pkg/sdk/session_policies_def.go

```diff
@@ -0,0 +1,110 @@
+package sdk
+
+import g "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/poc/generator"
+
+//go:generate go run ./poc/main.go
+
+var SessionPoliciesDef = g.NewInterface(
+	"SessionPolicies",
+	"SessionPolicy",
+	g.KindOfT[SchemaObjectIdentifier](),
+).
+	CreateOperation(
+		"https://docs.snowflake.com/en/sql-reference/sql/create-session-policy",
+		g.QueryStruct("CreateSessionPolicy").
+			Create().
+			OrReplace().
+			SQL("SESSION POLICY").
+			IfNotExists().
+			Name().
+			OptionalIntAssignment("SESSION_IDLE_TIMEOUT_MINS", g.ParameterOptions().NoQuotes()).
+			OptionalIntAssignment("SESSION_UI_IDLE_TIMEOUT_MINS", g.ParameterOptions().NoQuotes()).
+			OptionalTextAssignment("COMMENT", g.ParameterOptions().SingleQuotes()).
+			WithValidation(g.ValidIdentifier, "name").
+			WithValidation(g.ConflictingFields, "OrReplace", "IfNotExists"),
+	).
+	AlterOperation(
+		"https://docs.snowflake.com/en/sql-reference/sql/alter-session-policy",
+		g.QueryStruct("AlterSessionPolicy").
+			Alter().
+			SQL("SESSION POLICY").
+			IfExists().
+			Name().
+			Identifier("RenameTo", g.KindOfTPointer[SchemaObjectIdentifier](), g.IdentifierOptions().SQL("RENAME TO")).
```

Comment:
Should be OptionalIdentifier, but that also works

Reply:
Done.